DigiMeSDK/Core/Classes/DMEAuthorizationManager.m

```diff
@@ -78,6 +78,8 @@ - (void)handleAction:(DMEOpenAction *)action withParameters:(NSDictionary<NSStri
             self.authCompletionBlock(self.session, err);
         });
     }
+    
+    self.authInProgress = NO;
```

Comment:
I see - the reset back to false actually got lost in the refactor anyway! However, in the quickfix I meant to remove the `authInProgress` restriction when starting new auth. The use-case is:

When user is taken to digi.me app, they can press the home button, open up the 3rd party again and essentially be "stuck".

We might be able to just take out:

```objectivec
if (self.authInProgress)
    {
        NSError *authError = [NSError authError:AuthErrorInProgress];
        completion(self.session, authError);
        return;
    }
```

from `beginAuthorizationWithCompletion:` method

Reply:
As well as the error definition ))